components/identity-mgt/org.wso2.carbon.identity.mgt.admin.ui/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
re-factor pom using osgi best practices, import correct version, remove unused imports and dependencies.